orchestra/static/orchestra/common/js/orchestra.routes.js

```diff
@@ -30,7 +30,7 @@
         controllerAs: 'vm',
         templateUrl: $static('/static/orchestra/task/partials/task.html')
       })
-      .when('/project/:projectId', {
+      .when('/project/:projectId?', {
```

Comment:
If you go to the URL from the dashboard there's no project ID attached, so you get a default "select a project" screen